backend/app/services/rag/document_service.py

```
@@ -60,6 +104,9 @@ def _index_document_sync(
         Returns:
             Indexing result dict
         """
+        if file_path is None and attachment_id is None:
+            raise ValueError("Either file_path or attachment_id must be provided")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)